packages/store-cache/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@latticexyz/store-cache",
+  "version": "1.42.0",
+  "private": true,
+  "description": "Database to replicate Store contract state on a TypeScript client",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/latticexyz/mud.git",
+    "directory": "packages/store-cache"
+  },
+  "license": "MIT",
+  "type": "module",
+  "exports": {
+    ".": "./dist/index.js"
+  },
+  "types": "src/index.ts",
+  "scripts": {
+    "build": "tsup",
+    "clean": "rimraf dist",
+    "dev": "tsup --watch",
+    "release": "npm publish --access=public",
+    "test": "vitest --run --passWithNoTests"
+  },
+  "dependencies": {
+    "@latticexyz/common": "workspace:^",
```

Comment:
why `^` and not `*`?

Reply:
interesting, pnpm did this automatically when installing the common package